layouts/chromes/html5.php

```diff
@@ -23,7 +24,8 @@
 
 $moduleTag              = htmlspecialchars($params->get('module_tag', 'div'), ENT_QUOTES, 'UTF-8');
 $moduleAttribs          = [];
-$moduleAttribs['class'] = 'moduletable ' . htmlspecialchars($params->get('moduleclass_sfx'), ENT_QUOTES, 'UTF-8');
+$moduleAttribs['class'] = "$attribs[class] $module->position moduletable "
+			. htmlspecialchars($params->get('moduleclass_sfx'), ENT_QUOTES, 'UTF-8');
```

Comment:
```suggestion
                        . htmlspecialchars($params->get('moduleclass_sfx'), ENT_QUOTES, 'UTF-8');
```
@korenevskiy Please fix code style here. I will add back the test results after that so it will not require new tests.